Formula/maggot.rb

```diff
@@ -0,0 +1,27 @@
+class Maggot < Formula
+  include Language::Python::Virtualenv
+
+  desc "GUI frontend for Msc-generator"
+  homepage "https://sourceforge.net/p/msc-generator"
+  url "https://downloads.sourceforge.net/project/msc-generator/maggot/v1.x/maggot-1.3.3-py3-none-any.whl"
+  sha256 "98fda1ec7db109d6845ebe795c75d54a9fcba76ced1ea9ccf44b85b0c85a9065"
+  license "GPL-3.0-or-later"
+
+  depends_on "gtk+3"
+  depends_on "msc-generator"
+  depends_on "pango"
+  depends_on "pygobject3" # implicitly depends on the correct (brewed) python3 version
```

Comment:
I think you actually want to include `depends_on "python@3.9"` here, or else Homebrew `python3` will not be on `PATH` at buildtime.

This might also explain the missing library error.

Reply:
I removed it earlier today but will put it back.